nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -750,8 +750,8 @@ define([
             return;
         }
 
-        function doActionButton(data) {
-            switch (data.action) {
+        function doActionButton(action) {
```

Comment:
The object returned by the action button had changed but this had not been changed appropriately.